Bots/DotNet/Skills/CodeFirst/TeamsWaterfallSkillBot/Startup.cs

```diff
@@ -77,14 +81,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseDefaultFiles()
                 .UseStaticFiles()
+
+                //.UseWebSockets()
                 .UseRouting()
                 .UseAuthorization()
                 .UseEndpoints(endpoints =>
                 {
                     endpoints.MapControllers();
                 });
 
-            // app.UseHttpsRedirection();
+            // Uncomment this to support HTTPS.
```

Comment:
This comment seems out of place or refering to a line that is not there, consider removing it.